snappi_ixnetwork/trafficitem.py

```diff
@@ -690,10 +738,26 @@ def results(self, request):
             msg = "Invalid format of flow_names passed {},\
                     expected list".format(flow_names)
             raise Exception(msg)
-
-        filter = {'property': 'name', 'regex': '.*'}
-        filter['regex'] = '^(%s)$' % '|'.join(
-                        self._api.special_char(flow_names))
+        final_flow_names = [
+            f.name for f in self._api._config.flows
+            if f.metrics.enable is True and f.name in flow_names
```

Comment:
shouldn't we handle the case where `f.metrics` could be None ?

Reply:
done